runtime/src/lib.rs

```diff
@@ -1949,6 +1949,18 @@ impl hermes_governance_platform::Config for Runtime {
     type WeightInfo = hermes_governance_platform::weights::WeightInfo<Runtime>;
 }
 
+#[cfg(feature = "wip")] // order-book
+impl order_book::Config for Runtime {
+    const MAX_ORDER_LIFETIME: Moment = 2_592_000_000; // 30 days
```

Comment:
I think it's better to use predefined constants for it

Reply:
We have predefined constants for blocks per day. We have no something to have secs per day

Reply:
You can use `30 * (DAYS as Moment) * SECS_PER_BLOCK`. I think it looks better than much magic number